lib/whoisxmlapi2.rb

```diff
@@ -12,11 +12,20 @@
 require "whoisxmlapi2/version"
 
 module WhoisXMLAPI2
-  def self.configuration
-    @configuration ||= Configuration.new
-  end
+  class << self
+    def configuration(params = {})
+      @configuration ||= Configuration.new(params)
```

Comment:
Please add a README example documentation with code block for configuring this way